docker-compose.example.yml

```diff
@@ -9,11 +11,24 @@ services:
     volumes:
       - ./logs:/var/lib/server/logs
       - ./uploads:/opt/bundled/files/packages/files/dist/uploads
+    depends_on:
+      localstack:
+        condition: service_healthy
+      db:
+        condition: service_healthy
+      cache:
+        condition: service_healthy
     networks:
       - standardnotes_self_hosted
+    healthcheck:
+      test: curl -s http://localhost:3000
```

Comment:
This adds additional setup for the user to edit this port and address and we want to keep the editing of the example to a minimum

Reply:
Maybe change the internal port to 80 to make possible to remove the port, this make safe start and the container wait the correct time, avoid multiple common errors. What do you think, hard to change the internal port?

Reply:
80 is commonly taken by HTTP server applications

Reply:
well this is a api over http...

Reply:
Yes, but for example you might be setting up the self-hosted setup on a Ubuntu remote server which would have nginx or apache running. This way you'd end up conflicting the `80` port right off the bat.